docs/sphinx/features/device_allocators.rst

```diff
@@ -28,17 +28,19 @@ exceed the size that was used when making the device allocator.
 Retrieving a DeviceAllocator Object
 -----------------------------------
 
-The :class:`umpire::get_device_allocator` host/device function returns the DeviceAllocator object that corresponds 
+After creating a DeviceAllocator, we can immediately start using that allocator to allocate device memory. To do this, we
+have the :class:`umpire::get_device_allocator` host/device function which returns the DeviceAllocator object corresponding 
 to the ID or name given. The DeviceAllocator class also includes a helper function, :class:`umpire::is_device_allocator`,
 to query whether or not a given ID corresponds to an existing DeviceAllocator. Below is an example of using the **ID** to 
-obtain the DeviceAllocator object:
+obtain the first DeviceAllocator object:
 
 .. literalinclude:: ../../../examples/device-allocator.cpp
    :start-after: _sphinx_tag_get_dev_allocator_id_start
    :end-before: _sphinx_tag_get_dev_allocator_id_end
    :language: C++
 
-And next is an example of using the **name** instead:
+Note that the IDs start at one. The range of valid DeviceAllocator IDs is between 1 and 64, inclusively.
```

Comment:
I do not think that this should be exposed to the user.  They should always use is_device_allocator to determine whether it is a device allocator or not.  ID value semantics should not be exposed so that applications aren't tempted to make use of them.

Reply:
Even the fact that they start at one and not zero?